ParentsAndСhildren/ParentsAndСhildren/Parents.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace ParentsAndСhildren
+{
+    public class Parents
+    {
+        public string Name { get; private set; }
+
+        public Parents(string name)
+        {
+            Name = name;
+        }
+
+        public void PickUpChild(object sender, EventArgs e)
+        {
+            Child child = sender as Child;
+
+            Console.WriteLine($"{ Name } raised {child.Name}. " +
```

Comment:
It doesn't seem very obvious to me why a parent needs to know child's age to pick it up)), but the example of using class data is clear. By the way, maybe parents could use this information making the decision whether to help their child or not. Kind of at certain age children are grown-up enough to do it on their own.